docs/GettingStartedGuide.md

```diff
@@ -56,4 +62,14 @@ More details how to use the CatalogClient are placed in the [Catalog usage examp
 
 ## <a name="examples"></a> Layer Usage Example
 
-[Versioned Layer Read example](./examples/nodejs-read-versioned-layer.md) shows how to use VersionLayerClient and how to retrieve data, metadata and partition data from versioned layer, using OLP SDK for TypeScript.
\ No newline at end of file
+[Versioned Layer Read example](./examples/nodejs-read-versioned-layer.md) shows how to use VersionLayerClient and how to retrieve data, metadata and partition data from versioned layer, using OLP SDK for TypeScript.
```

Comment:
```suggestion
[Versioned Layer Read example](./examples/nodejs-read-versioned-layer.md) shows how to use the VersionLayerClient and how to retrieve data, metadata and partition data from a OLP versioned layer, using OLP SDK for TypeScript.
```

What is the difference between data and partition data in the above sentence?
Probably you mean layer metadata, partition metadata and partition data.

Reply:
It is double. I've removed it. Correct example message is on the line #51